operator/internal/manifests/var.go

```diff
@@ -19,6 +19,8 @@ const (
 	grpcPort         = 9095
 	protocolTCP      = "TCP"
 
+	gossipBindAddrEnvVarName = "POD_IP"
```

Comment:
I'd probably prefer the environment variable being named after the "purpose" and not the "content". I know that it contains the "pod IP", but we are using it to set a bind address (or maybe an advertised address). That should make it clearer in debug output.